tests/integration/security/util/cert/cert.go

```diff
@@ -182,6 +182,10 @@ func CreateCustomEgressSecret(ctx resource.Context) error {
 
 	_, err = kubeAccessor.CoreV1().Secrets(systemNs.Name()).Create(context.TODO(), secret, metav1.CreateOptions{})
 	if err != nil {
+		if errors.IsAlreadyExists(err) {
```

Comment:
just improve to allow easier running locally when we run multiple tests in a row